initializer/process_resource.py

```diff
@@ -0,0 +1,115 @@
+import os
```

Comment:
I think we should include some copyright or notice for this code that we copied from otel